_updates/2021-07-14.html

```diff
@@ -2,17 +2,38 @@
 ---
 <div class="ds-u-padding-y--2">
   <div class="ds-u-font-size--h3 ds-u-font-weight--bold">
-    Version 2 of BCDA is now live, conforming with the FHIR R4 specification!
+    Version 1 of the BCDA Partially Adjudicated Claims Enhancement live for use on May 16!
   </div>
   <div class="ds-u-font-size--lead ds-u-padding-y--1">
-    July 14, 2021
+    May 16, 2023
```

Comment:
there are two May 16th updates:

1. "Version 1 of the BCDA Partially Adjudicated Claims Enhancement is now live for production use."
2. "Version 1 of the BCDA Partially Adjudicated Claims Enhancement live for use on May 16!"

It looks like the wireframe only specifies one; has Ali mentioned keeping both?

Reply:
Ali said wireframe is final draft